linear_programming/simplex.py

```diff
@@ -62,48 +59,35 @@ def __init__(self, tableau: np.ndarray, n_vars: int) -> None:
         # Does objective row only contain (non)-negative values?
         self.stop_iter = False
 
-    @staticmethod
-    def generate_col_titles(*args: int) -> list[str]:
+    def generate_col_titles(self) -> list[str]:
         """Generate column titles for tableau of specific dimensions
 
-        >>> Tableau.generate_col_titles(2, 3, 1)
-        ['x1', 'x2', 's1', 's2', 's3', 'a1', 'RHS']
-
-        >>> Tableau.generate_col_titles()
-        Traceback (most recent call last):
-        ...
-        ValueError: Must provide n_vars, n_slack, and n_art_vars
-        >>> Tableau.generate_col_titles(-2, 3, 1)
-        Traceback (most recent call last):
-        ...
-        ValueError: All arguments must be non-negative integers
```

Comment:
Do we NEED to remove these tests?!?

Reply:
generate_col_titles was no longer a static method. I added another doctest for Tableau class, however